janitor/functions.py

```diff
@@ -27,6 +28,95 @@
 )
 
 
+def unionize_dataframe_categories(
+    *dataframes, column_names: Union[str, Iterable[str], Any] = None
+) -> List[pd.DataFrame]:
+    """
+    Given a group of dataframes which contain some categorical columns, for
```

Comment:
Minor nitpick on this, @zbarry, but could we start using [semantic line breaks](https://sembr.org) for the docstrings? Yes, there will be more lines, but I think it'll make diffing and editing much easier (as stated in the `sembr` page.

Reply:
This is awesome! I never realized I was missing this functionality

Reply:
No prob. I think it conforms to the semantic line break protocol now.